pkg/asset/installconfig/vsphere/vsphere.go

```diff
@@ -170,6 +178,36 @@ func getClients() (*vCenterClient, error) {
 	}, nil
 }
 
+// checkComputeResource checks the ClusterComputeResource to see if there are any attached hosts.  Without
+// hosts, the check for networks will fail.  We want the message to be clear if this is the cause so
+// users know what to address.  This only returns an error if unable to verify or if no ComputeResources
+// are found.
+func checkComputeResource(ctx context.Context, datacenter string, cluster string, finder Finder) error {
```

Comment:
Typically for validation that resources exist and/or are valid, we perform that check in the validation [validation phase](https://github.com/openshift/installer/blob/master/pkg/asset/installconfig/vsphere/validation.go).  We could probably add it to https://github.com/openshift/installer/blob/master/pkg/asset/installconfig/vsphere/validation.go#L347.

Reply:
Ok, I'll have to look into moving it over there.  If we do this, is it still valid to use it as part of the automated install-config creation steps?